c++/src/capnp/schema.capnp

```diff
@@ -170,6 +170,9 @@ struct Node {
     }
   }
 
+  startByte @34 :UInt32;
```

Comment:
I'd argue these belong in `SourceInfo` -- in fact, it has a TODO for this.

Reply:
@kentonv 

I'm sorry.
I moved byte position definition to ``SourceInfo``.